packages/web/src/scss/components/UNSTABLE_ProductLogo/_UNSTABLE_ProductLogo.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:map';
+@use 'theme' as theme;
+@use '../../tools/breakpoint';
+
+.UNSTABLE_ProductLogo {
+    width: fit-content;
```

Comment:
I think that `width: fit-content;` a little bit replaces `display` property (probably because of the `Stack` component that uses `flex`)?

Not sure how this property cooperate with `flex` or `grid` parent, but wouldn't be more safe to use well known `display: inline-block;`?

Reply:
There is a question whether the width of the container should be adapted to the width of the logo (which I do not know beforehand) or whether it should be stretched to the width of the parent element. This solution ensures that the width of the container fits the width of the logo - other solutions don't work for me, including the one you suggested

Reply:
Do not forget that `img` tag will most likely have `width` and `height` HTML attributes, i.e. when using NextJS's component `Image`, those properties are required and will be rendered.

Reply:
@dlouhak sorry itś work with display: inline-block. I have wrong stack alignment in demo (stretch). Updated.

@vmilersky As I have already responded to another comment, I am aware of this, but I honestly do not see any problem that would prevent correct use 🤷‍♂️